4634_Configs/vdmDriverII_Config_PCC_4634.json

```diff
@@ -0,0 +1,126 @@
+{
+"Fill": "4634",
+"Date": "Nov192015",
+"Luminometer": "PCC",
+"AnalysisDir": "Fill4634_Nov192015",
+"CorrLater" : ["LengthScale", "Ghosts", "Satellites"],
+"Corr" : ["noCorr"],
+"Comment": "Output of following step goes to <AnalysisDir>/cond", 
+"makeScanFile": false,
+"Comment": "Output of following step goes to <AnalysisDir>/LuminometerData",
+"makeRateFile": true,
+"Comment": "Output of following step goes to <AnalysisDir>/cond", 
+"makeBeamCurrentFile": false,
+"Comment": "Output of following step goes to <AnalysisDir>/corr",
+"makeBeamBeamFile": false,
+"Comment": "Output of following step goes to <AnalysisDir>/corr",
+"makeGhostsFile": false,
+"Comment": "Output of following step goes to <AnalysisDir>/corr",
+"makeSatellitesFile": false,
+"Comment": "Output of following step goes to <AnalysisDir>/<Luminometer>/graphs", 
+"makeLengthScaleFile": false,
+"Comment": "Output of following step goes to <AnalysisDir>/corr",
+"makeGraphsFile": false,
+"Comment": "Output of following step goes to <AnalysisDir>/corr",
+"makeGraphs2D": false,
+"Comment": "Output of following step goes to <AnalysisDir>/<Luminometer>/results/<Corr>", 
+"runVdmFitter": false,
+"makeScanFileConfig":
+	{
+	"InputCentralPath": "/afs/cern.ch/user/g/gkrintir/tempeos/cms/store/group/comm_luminosity/VdM/scanFill4634_151119/central2/localdata/central18/",
+	"InputDIPFile" : "/afs/cern.ch/user/g/gkrintir/tempeos/cms/store/group/comm_luminosity/VdM/scanFill4634_151119/vdm/vdm_Fill4634.csv",
+	"ScanNames" : ["X1","Y1","Y2","X2","Y3","X3","IY1","IX1","IY2","IX2"],
+	"Comment" : "Since X,Y scans are automatically differentiated, a time window can include an X and a Y scan, will still work, timestamp in UTC", 
+	"ScanTimeWindows" :[[1447946522, 1447947608],[1447947870, 1447948900],[1447949003, 1447950033],[1447950098, 1447951128],[1447963841, 1447964820],[1447964893, 1447965872],[1447966215,1447966762],[1447966840, 1447967392],[1447967567, 1447968120],[1447968195, 1447968743]],
+	"Comment" : "A few parameters that are not in our current DIP file, but should be available eventually",
+	"Comment" : "betaStar in cm, angle in microrad",
+	"BetaStar" : 400.0,
```

Comment:
Please change the comment above to "betaStar in m" and following line to
"BetaStar"  :  4.0,